include/smack/SmackRep.h

```diff
@@ -94,6 +108,11 @@ class SmackRep {
   const Expr* b2i(const llvm::Value* v);
 
 public:
+  void useBitVector();
```

Comment:
These four functions should also be removed, preferring the use of `SmackOptions::BitVectors` and `SmackOptions::InferFieldOverlap`.